src/reader/DocReader.java

```diff
@@ -0,0 +1,47 @@
+package reader;
+
+import java.io.*;
+import java.util.*;
+
+public class DocReader extends IReader {
+
+    private final String filesPath;
+
+    public DocReader(String filesPath) {
+        super();
+        this.filesPath = filesPath;
+    }
+
+    private File[] getFiles() {
+        File directoryPath = new File(filesPath);
+        return directoryPath.listFiles();
+    }
+
+    private StringBuilder getFileLines(File file) {
```

Comment:
There is a class with name StringBuilder in java, please rename this.(it also should be in lamelCase format)